index.js

```diff
@@ -0,0 +1,422 @@
+  { which: 87, code: "KeyW", en: "w", ru: "ц" },
+  { which: 69, code: "KeyE", en: "e", ru: "у" },
+  { which: 82, code: "KeyR", en: "r", ru: "к" },
+  { which: 84, code: "KeyT", en: "t", ru: "е" },
+  { which: 89, code: "KeyY", en: "y", ru: "н" },
+  { which: 85, code: "KeyU", en: "u", ru: "г" },
+  { which: 73, code: "KeyI", en: "i", ru: "ш" },
+  { which: 79, code: "KeyO", en: "o", ru: "щ" },
+  { which: 80, code: "KeyP", en: "p", ru: "з" },
+  {
+    which: 219,
+    code: "BracketLeft",
+    en: "[",
+    ru: "х",
+    shift: "{",
+    ruShift: "Х",
+  },
+  {
+    which: 221,
+    code: "BracketRight",
+    en: "]",
+    ru: "ъ",
+    shift: "}",
+    ruShift: "Ъ",
+  },
+  { which: 220, code: "Backslash", en: "\\", shift: "|", ruShift: "/" },
+
+  { which: 20, code: "CapsLock", en: "CapsLock" },
+  { which: 65, code: "KeyA", en: "a", ru: "ф" },
+  { which: 83, code: "KeyS", en: "s", ru: "ы" },
+  { which: 68, code: "KeyD", en: "d", ru: "в" },
+  { which: 70, code: "KeyF", en: "f", ru: "а" },
+  { which: 71, code: "KeyG", en: "g", ru: "п" },
+  { which: 72, code: "KeyH", en: "h", ru: "р" },
+  { which: 74, code: "KeyJ", en: "j", ru: "о" },
+  { which: 75, code: "KeyK", en: "k", ru: "л" },
+  { which: 76, code: "KeyL", en: "l", ru: "д" },
+  { which: 186, code: "Semicolon", en: ";", ru: "ж", shift: ":", ruShift: "Ж" },
+  { which: 222, code: "Quote", en: "'", ru: "э", shift: '"', ruShift: "Э" },
+  { which: 13, code: "Enter", en: "Enter" },
+
+  { which: 16, code: "ShiftLeft", location: 1, en: "Shift" },
+  { which: 90, code: "KeyZ", en: "z", ru: "я" },
+  { which: 88, code: "KeyX", en: "x", ru: "ч" },
+  { which: 67, code: "KeyC", en: "c", ru: "с" },
+  { which: 86, code: "KeyV", en: "v", ru: "м" },
+  { which: 66, code: "KeyB", en: "b", ru: "и" },
+  { which: 78, code: "KeyN", en: "n", ru: "т" },
+  { which: 77, code: "KeyM", en: "m", ru: "ь" },
+  { which: 188, code: "Comma", en: ",", ru: "б", shift: "<", ruShift: "Б" },
+  { which: 190, code: "Period", en: ".", ru: "ю", shift: ">", ruShift: "Ю" },
+  { which: 191, code: "Slash", en: "/", ru: ".", shift: "?", ruShift: "," },
+  { which: 38, code: "ArrowUp", en: "▲" },
+  { which: 16, code: "ShiftRight", location: 2, en: "Shift" },
+
+  { which: 17, code: "ControlLeft", location: 1, en: "Ctrl" },
+  { which: 18, code: "AltLeft", location: 1, en: "Alt" },
+  { which: 32, code: "Space", en: " " },
+  { which: 18, code: "AltRight", location: 2, en: "Alt" },
+  { which: 17, code: "ControlRight", location: 2, en: "Ctrl" },
+  { which: 37, code: "ArrowLeft", en: "◄" },
+  { which: 40, code: "ArrowDown", en: "▼" },
+  { which: 39, code: "ArrowRight", en: "►" },
+];
+
+class Button {
+  constructor(obj) {
+    this.which = obj.which;
+    this.code = obj.code;
+    this.location = obj.location || 0;
+    this.en = obj.en;
+    this.ru = obj.ru || obj.en;
+    this.shift = obj.shift;
+    this.ruShift = obj.ruShift || obj.shift;
+  }
+  create = () => {
```

Comment:
`Arrow` functions is really great feature, but sometimes you have to prefer usual function declaration, especially in classes. Why? In this case, `create` is just a field of object that happens to have a function as its value, and it doesn't participate in `prototype chain`. I know it may sound difficult, and there is rule of thumb:

If you don't know what type of function to use in a class, use function declaration, i.e.:
```
class Some {
   method() {...}
}
```

😺